Azure/Prod/resource_sql.tf

```diff
@@ -48,6 +54,7 @@ resource "azurerm_sql_firewall_rule" "bc-sql-sec" {
   end_ip_address      = "0.0.0.0"
 }
 
+#SQL Database using for the BuyingCatalgueService Private
```

Comment:
Catalogue (not going to repeat this comment - looks like a copy/paste error everywhere